option.go

```diff
@@ -30,3 +31,9 @@ func WithNewLine() Option {
 		opt.beforeNewline = true
 	}
 }
+
+func WithReadFallbackToRawString() Option {
```

Comment:
I generally agree with the code comment left above that we should probably split these into operation specific options, but I think that's out of scope for this PR.